components/conjunctions/_screen-sm-min.scss

```diff
@@ -13,14 +13,14 @@
 		flex-direction: row;
 
 		&:not(.brdr-0) {
-			[class*="cnjnctn-col"] {
+			> [class*="cnjnctn-col"] {
```

Comment:
Since there is only one rule here, move sub rule to parent rule. Do this for line 11 also.